uk.co.compucorp.civicrm.hrleaveandabsences/views/manager-leave/components/manager-leave-calendar.html

```diff
@@ -1,796 +1,243 @@
         </div>
+      </div>
+      <div class="panel-subheading">
         <div class="row">
-          <div class="col-sm-3">
-            <div class="chr_leave-calendar__legend__entry">
-              <div class="chr_leave-calendar__item chr_leave-calendar__item--type--weekend"></div>
-              <span class="chr_leave-calendar__legend__label">Weekend</span>
-            </div>
-          </div>
-          <div class="col-sm-3">
-            <div class="chr_leave-calendar__legend__entry">
-              <div class="chr_leave-calendar__item chr_leave-calendar__item--type--public-holiday"></div>
-              <span class="chr_leave-calendar__legend__label">Public Holiday</span>
+          <div class="col-sm-5">
+            <ui-select multiple
+                       limit="4"
+                       ng-model="calendar.selectedMonths">
+              <ui-select-match placeholder="Select Months">
+                <span ng-bind="$item"></span>
+              </ui-select-match>
+              <ui-select-choices repeat="month in (calendar.months | filter: $select.search)">
+                <span ng-bind="month"></span>
+              </ui-select-choices>
+            </ui-select>
+          </div>
+          <div class="col-sm-offset-4 col-sm-3">
+            <div class="chr_custom-select chr_custom-select--full">
+              <select
+                ng-options="period as calendar.labelPeriod(period) for period in calendar.absencePeriods track by period.id"
+                ng-model="calendar.selectedPeriod"
+                ng-change="calendar.refresh()">
+              </select>
             </div>
           </div>
-          <div class="col-sm-3">
-            <div class="chr_leave-calendar__legend__entry">
-              <div class="chr_leave-calendar__item chr_leave-calendar__item--type--non-working"></div>
-              <span class="chr_leave-calendar__legend__label">Non Working Day</span>
+        </div>
+      </div>
+    </div>
+
+    <crm-loading show="!calendar.loading.calendar">
+      <section class="panel panel-default"
+               ng-repeat="month in (calendar.selectedMonths.length > 0 ? calendar.selectedMonths : calendar.months)">
+        <header class="panel-heading">
+          <h2 class="panel-title">{{month}}</h2>
+        </header>
+        <div class="table-responsive">
+          <div
+            class="text-center"
+            ng-show="calendar.filterContacts().length === 0">
+            Sorry No Contacts
+          </div>
+          <table
+            ng-hide="calendar.filterContacts().length === 0"
+            class="table table-bordered chr_leave-calendar">
+            <thead>
+            <tr class="chr_leave-calendar__days">
+              <th class="chr_leave-calendar__name">Employee Name</th>
+              <!--send any contact id to get the days for a month-->
+              <th ng-repeat="day in calendar.getMonthData(calendar.managedContacts[0].id, calendar.months.indexOf(month))">
```

Comment:
This is used to only create the month header,
![manager_leave___hr17](https://cloud.githubusercontent.com/assets/5058867/23887492/40d67e58-08a8-11e7-875f-c1a5648b2970.png)
So it does not matter which contact I am using.